docs/quick-start.md

```diff
@@ -0,0 +1,130 @@
+## 1. Prerequisites
+
+Before deploying the Ceph-CSI-Operator, ensure the following requirements are met:
+
+- A Kubernetes cluster ([supported version](https://kubernetes.io/releases/) recommended)
+- Ceph cluster ([supported version](https://docs.ceph.com/en/latest/releases/) recommended)
+- `kubectl` CLI installed
+
+**Note:** In this guide, we will use minimal configurations to deploy the Ceph-CSI-Operator and drivers. You can customize the configurations as per your requirements.
+
+## 2. Install the Ceph-CSI Operator
+
+```console
+kubectl create -f deploy/all-in-one/install.yaml
+```
+
+verify the installation:
+
+```bash
+kubectl get pods -n ceph-csi-operator-system
+NAME                                                    READY   STATUS    RESTARTS   AGE
+ceph-csi-operator-controller-manager-67d45fd9ff-zgst7   2/2     Running   0          40s
+```
+
+### 3. Create OperatorConfig CR
+
+Create an OperatorConfig CR to configure the Ceph-CSI Operator:
+
+```console
+echo '
+apiVersion: csi.ceph.io/v1alpha1
+kind: OperatorConfig
+metadata:
+  name: ceph-csi-operator-config
+  namespace: ceph-csi-operator-system
+' | kubectl create -f -
+```
+
+### 4. Deploy Ceph-CSI Drivers
+
+Once the operator is installed and configured, deploy the Ceph-CSI drivers:
+
+#### 4.1 Deploy the RBD Driver
+
+```console
+echo '
+apiVersion: csi.ceph.io/v1alpha1
+kind: Driver
+metadata:
+  name: rbd.csi.ceph.com
+  namespace: ceph-csi-operator-system
+' | kubectl create -f -
+```
+
+#### 4.2 Deploy the CephFS Driver
+
+```console
+echo '
+apiVersion: csi.ceph.io/v1alpha1
+kind: Driver
+metadata:
+  name: cephfs.csi.ceph.com
+  namespace: ceph-csi-operator-system
+' | kubectl create -f -
+```
+
+### 4.3 Deploy the Ceph-NFS Driver
+
+```console
+echo '
+apiVersion: csi.ceph.io/v1alpha1
+kind: Driver
+metadata:
+  name: nfs.csi.ceph.com
+  namespace: ceph-csi-operator-system
+' | kubectl create -f -
+```
+
+## 5. Verify Installation
+
+To verify the installation, check the status of the Ceph-CSI components:
+
+```bash
+kubectl get pod -nceph-csi-operator-system
+NAME                                                    READY   STATUS    RESTARTS   AGE
+ceph-csi-operator-controller-manager-744dc99cb5-scxxh   2/2     Running   0          45s
+cephfs.csi.ceph.com-ctrlplugin-5847c998b5-xf85m         5/5     Running   0          27s
+cephfs.csi.ceph.com-nodeplugin-r6pkt                    2/2     Running   0          27s
+nfs.csi.ceph.com-ctrlplugin-76fd4f5b4c-smk2g            5/5     Running   0          27s
+nfs.csi.ceph.com-nodeplugin-kbzms                       2/2     Running   0          27s
+rbd.csi.ceph.com-ctrlplugin-6965dcfdb8-w88kn            5/5     Running   0          4m35s
+rbd.csi.ceph.com-nodeplugin-lnm4n                       2/2     Running   0          4m35s
+```
+
+## 6. Clean Up Resources
+
+To clean up the resources, delete the OperatorConfig CR, drivers:
+
+```console
+kubectl delete operatorconfig ceph-csi-operator-config -n ceph-csi-operator-system
```

Comment:
Continued from the last comment, for a minimal installation example we don't need an operator config CR